tests/test_fitsCompression.py

```diff
@@ -56,7 +56,7 @@ def checkAstropy(image, filename, hduNum=0):
     hduNum : `int`
         HDU number of interest.
     """
-    print("Astropy currently doesn't read our compressed images perfectly.")
+#    print("Astropy currently doesn't read our compressed images perfectly.")
```

Comment:
This line may not be good as a `print`, but commenting it out seems worse.

Reply:
Agreed. That was accidental (the voluminous output was annoying and slowed down a test I was running frequently)